data/yaml/venues.yaml

```diff
@@ -221,6 +221,10 @@ coling:
 comacoma:
   acronym: ComAComA
   name: Workshop on Computational Approaches to Compound Analysis
+computel:
+  acronym: ComputEL
+  name: The 4th Workshop on the Use of Computational Methods in the Study of Endangered
+    Languages Volume 1 (Papers)
```

Comment:
The title here should be the general workshop name, not a particular meeting of the venue. This appears to be a proceedings volume name.

Also please add `url: https://www.computel-workshop.org/`